pkg/resource/db_cluster/conditions.go

```diff
@@ -0,0 +1,102 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is this file now covered as part of the latest conditions changes in runtime?

Reply:
nope. will do in a later commit.

Reply:
@RedbackThomson if no other concerns, may we go forward with this PR? I was hoping to cut a new release of RDS controller today.

Reply:
Yeah it all looks solid to me.